src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -1791,10 +1790,13 @@ private static HttpClient CreateHttpClient(IWebProxy webProxy)
 
         private static bool TrySetBaseUrl(
             string baseUrl,
-            string fallbackBaseUrl,
             out string target)
         {
-            if (!Uri.TryCreate(baseUrl, UriKind.Absolute, out var baseUri))
+            if (baseUrl is null) throw new ArgumentNullException(nameof(baseUrl));
+
+            if (!Uri.TryCreate(baseUrl, UriKind.Absolute, out var baseUri) ||
+                string.IsNullOrEmpty(baseUri.Scheme) ||
+                string.IsNullOrEmpty(baseUri.Authority))
```

Comment:
This has broken the bot since `16.0.0-ci.210`. I'm not setting any custom base URL, but the exception is always thrown, preventing the bot from starting.